swagger_server/controllers/operand_controller.py

```diff
@@ -1,15 +1,27 @@
+import flask
+
 from swagger_server import util
+from swagger_server.models.evaluation import Operand
+from swagger_server.util import ValidationError
 
 
-def add_operand():  # noqa: E501
+def add_operand(payload):  # noqa: E501
     """Create an operand
 
     Creates an operand # noqa: E501
 
 
     :rtype: None
     """
-    return 'do some magic!'
+    if not payload:
+        raise ValidationError(400, 'Couldn\'t parse JSON POST body.')
+    if not payload.get('id'):
```

Comment:
I think you don't need to check for `id` because that's an autoincrement but you could validate `expression_id`, `rank`, `value` and `type`